packages/shared/src/graphql/squads.ts

```diff
@@ -662,14 +662,12 @@ export const SQUAD_MODERATE_POST_MUTATION = gql`
   mutation ModerateSourcePost(
     $postIds: [ID]!
     $status: String
-    $sourceId: ID!
```

Comment:
We have to be careful with deployment. If we are updating the existing mutation, we must merge this PR at the same time the API is deployed. Otherwise, requests might fail.